### PR TITLE
CI: Make alert_pending_release.sh filter substrate changes based on github labels...

### PR DESCRIPTION
**Pull request by s3krit**
... and split common functions into lib.sh.

### CODE REVIEW

File: scripts/gitlab/lib.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```
Comment (bkchr) [inline scripts/gitlab/lib.sh]:
```suggestion
#!/usr/bin/env bash
```

`/bin/bash` is not a standard path and should never be used!

File: scripts/gitlab/alert_pending_release.sh [hunk 2]
```diff
@@ -1,19 +1,7 @@
 #!/bin/bash
```
Comment (bkchr) [inline scripts/gitlab/alert_pending_release.sh]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/gitlab/publish_draft_release.sh [hunk 3]
```diff
@@ -1,55 +1,19 @@
 #!/bin/bash
```
Comment (bkchr) [inline scripts/gitlab/publish_draft_release.sh]:
```suggestion
#!/usr/bin/env bash
```

File: scripts/gitlab/lib.sh (hunk 1 above)
Comment (s3krit) [reply scripts/gitlab/lib.sh]: In the context of where these scripts run (a docker image we maintain), it's definitely going to be /bin/bash but I agree with the general case. Will fix :)

**Comment by TriplEight** [comment]
*Clapping*

**Pull request merged**